### PR TITLE
Adds Flowthings's browser client to jsdelivr

### PR DESCRIPTION
**Pull request by gmmeyer**

### CODE REVIEW

File: files/flowthings/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "npm",
+  "name": "flowthings-browser",
+  "repo": "flowthings/browser-client",
+  "files": {
+    "basePath": "dist/",
+    "include": ["flowthings-browser.js", "flowthings-browser.min.js"]
```
Comment (jimaek) [inline files/flowthings/update.json]: I dont think that `flowthings-browser.js` is needed in production. Can you remove it?

**Comment by gmmeyer** [comment]
yes

**Comment by gmmeyer** [comment]
It's removed! :)

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
Stupid robots disobeying their masters

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
@gmmeyer Thanks for the PR

**Pull request closed without merging**